atlasdb-workload-server-api/src/main/java/com/palantir/atlasdb/workload/transaction/WriteTransactionAction.java

```diff
@@ -34,6 +35,19 @@ public interface WriteTransactionAction extends TransactionAction {
     @Value.Parameter
     Integer value();
 
+    @Value.Derived
+    default WorkloadCell indexCell() {
```

Comment:
This will also change later on, but I think it's OK to keep for now. Even when we switch writes to be a list, we could always make this a static utility.